ramls/resources.raml

```diff
@@ -86,50 +24,53 @@ types:
-            required: true
-            properties:
-              type: string
-              attributes:
-                description: Needed because of JSON API
-                type: object
-                required: true
-                properties:
-                  packageId:
-                    description: |
-                      Id of the custom package to which the managed/custom title is to be associated.
-                      Note that packageId is a combination of vendorId-packageId.
-                    type: string
-                    example: 123355-2845510
-                    required: true
-                  titleId:
-                    description: Id of the managed/custom title that needs to be associated to a custom package.
-                    type: string
-                    example: "17059786"
-                    required: true
-                  url:
-                    description: Custom URL displaying the relationship between the custom package and custom/managed title.
-                    type: string
-                    required: false
-                    example: https://hello.io
-        example: !include examples/resources/resources_post_request.json
+        description: The server has successfully fulfilled the POST request.
+        type: resourcePostRequest
+        example:
+          strict: false
+          value: !include examples/resources/resources_post_request.json
     responses:
       200:
         description: OK
         body:
           application/vnd.api+json:
-            example: !include examples/resources/resources_post_200_response.json
+            description: The server has successfully fulfilled the POST request.
+            type: resource
+            example:
+              strict: false
+              value: !include examples/resources/resources_post_200_response.json
       400:
         description: Bad Request
+        body:
+          application/vnd.api+json:
+            type: jsonapiError
+            example:
+              strict: false
+              value: !include examples/resources/resources_post_400_response.json
+      404:
+        description: Not Fount
+        body:
+          application/vnd.api+json:
+            type: jsonapiError
+            example:
+              strict: false
+              value: !include examples/resources/resources_post_404_response.json
       422:
         description: Unprocessable Entity
         body:
           application/vnd.api+json:
-            example: !include examples/resources/resources_post_422_response.json
+            type: jsonapiError
+            example:
+              strict: false
+              value: !include examples/resources/resources_post_422_response.json
   /{resourceId}:
+    displayName: Get resource by resourceId
+    description: Instance of a resource given resourceId
+    uriParameters:
+      resourceId:
+        description: Identifier of the resource to get
```

Comment:
Can we add to the description that "resourceId is a combination of providerId-packageId-titleId"? Thanks!

Reply:
thanks, updated